homeassistant/components/tuya/number.py

```diff
@@ -382,20 +382,18 @@ def __init__(
                 return
 
             uoms = DEVICE_CLASS_UNITS[self.device_class]
-            self._uom = uoms.get(self.native_unit_of_measurement) or uoms.get(
+            uom = uoms.get(self.native_unit_of_measurement) or uoms.get(
```

Comment:
Ha!
This is not covered in tests, but in `number.py` I'm only converting a class variable to a local variable